### DISCUSSION TITLE
CONCUR_READ_ONLY not supported for given type

### DISCUSSION THREAD

**Discussion by kothapraveen** [FreeTDS/freetds#540] [Q&A]
Hi @freddy77 

Thanks for your support.

Currently am facing below error, when we are working with Unix ODBC & FreeTDS.

**CONCUR_READ_ONLY not supported for given resultset type TYPE_SCROLL_INSENSITIVE 
but its working fine when the resultset type is TYPE_FORWARD_ONLY**

**Below are ResultSetTypes:**
TYPE_FORWARD_ONLY
TYPE_SCROLL_INSENSITIVE
TYPE_SCROLL_SENSITIVE

**Below are ResultSetConcurrency**
CONCUR_READ_ONLY
CONCUR_UPDATABLE

**Comment by freddy77** [comment]
Which query are you trying to execute ?
